plans/super/163-drafter-business-rules-fidelity.md

```
@@ -0,0 +1,237 @@
+```sql
+SELECT trip_id, duration_minutes FROM {{ this }} WHERE duration_minutes <= 0
+```
+
+Pipeline completed cleanly (exit 0, audits intact, cost rollup unaffected). The failure is the drafter's instruction-following on `business_rules` — not anything SignalForge's own code paths control directly.
+
+## Discovery findings
+
+### Current rendering shape (verified in code)
+
+- `src/signalforge/draft/prompts.py:557-579` — `_render_business_rules_section(model)` renders rules as a plain bulleted list under `## BUSINESS RULES` in the **dynamic block**:
+  ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)